e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -616,7 +616,7 @@ describeEE("scenarios > embedding > dashboard appearance", () => {
         .findByTestId("embed-frame")
         .invoke("attr", "data-embed-theme")
         .then(embedTheme => {
-          expect(embedTheme).to.eq(undefined); // default value, unset
```

Comment:
The combo of ensuring a default value for theme (i.e. `DisplayTheme` is not null or undefined) and the changes to `EmbedFrame` will set `data-embed-theme` to "light"